doc/tools/coqrst/notations/fontsupport.py

```diff
@@ -12,6 +12,9 @@
 """Transform a font to center each of its characters in square bounding boxes.
 
 See https://stackoverflow.com/questions/37377476/ for background information.
+
+This script is there for reference. It was used to generate the modified
+font UbuntuMono-B.ttf.
```

Comment:
`s/here/there/`, but more importantly it was used to generate `CoqNotations.ttf` from `UbuntuMono-B.ttf` :)

Reply:
Ah OK, thanks.